<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QFrame" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="CNX_StatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUDIOFOCUS_TRANSIENT">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>AudioFocus
Transient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUDIOFOCUS_LOSS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>AudioFocus
Loss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_VIDEOFOCUS_LOSS">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>VideoFocus
Loss</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_VIDEO_PRIORITY_NORMAL">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BUTTON_SEND_MESSAGE">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_CHECK_FOCUS">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Focus</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_VIDEO_PRIORITY_HIGHEST">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Highest</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="RADIO_AUDIO_PRIORITY_HIGH">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BUTTON_VIDEOFOCUS_TRANSIENT">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>VideoFocus
Transient</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_AUDIO_PRIORITY_NORMAL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUDIOFOCUS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>AudioFocus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_VIDEOFOCUS">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>VideoFocus</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RADIO_VIDEO_PRIORITY_HIGH">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BUTTON_POPUP_MESSAGE">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>150</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Popup Message</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>When audio focus is lost</string>
   </property>
   <widget class="QRadioButton" name="RADIO_STAY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stay</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RADIO_TERMINATE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminate</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CNX_StatusBar</class>
   <extends>QFrame</extends>
   <header location="global">CNX_StatusBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
